### DISCUSSION TITLE
Zoom Group Practical 3

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#445] [Old]
Hi Zoom members,

Here's a discussion to share your repositories for practical series 3.

So far I think only Sreedevi's package is up
(You can find it here: https://github.com/IBAHCM/SreedeviGummuluriSeries03)

Mine is up too (https://github.com/IBAHCM/ISchwandnerSeries03). But there are a lot of issues with it still. I've listed some of them under issues. If you have the time to take a look, I would be super grateful, but I appreciate that listing all the issues, would be quite time consuming. Whoever can find all the issues or any issues I haven't listed gets a jar full of my tears.

Let us know when your repositories are up for checking. (I will try to tag the members in the comments, so far it hasn't quite worked).

Best, Imo

**Comment by sreedevi-gummuluri** [comment]
hi Imo,
I think you already created a discussion group for series 3, just FYI:
https://github.com/IBAHCM/RPiR/discussions/439

This is the one I've listed for my practical 3 submission. 

Thank you for this additional link and listing my repository path.

Best,
Sreedevi

**Reply by ISchwandner** [reply]
Oh my, thanks for flagging that up! I'm a bit in over my head just now.

**Reply by sreedevi-gummuluri** [reply]
no worries :)

**Reply by maryadejoh** [reply]
Just wanted to note that too. Thanks for bringing this to light.

Cheers.
